frontend/src/component/feature/FeatureStrategy/FeatureStrategyForm/FeatureStrategyForm.tsx

```diff
@@ -350,19 +349,8 @@ export const FeatureStrategyForm = ({
             <StyledHeaderBox>
                 <StyledTitle>
                     {formatStrategyName(strategy.name || '')}
-                    <ConditionallyRender
-                        condition={
-                            !addEditStrategy &&
-                            strategy.name === 'flexibleRollout'
-                        }
-                        show={
-                            <Badge color='success' sx={{ marginLeft: '1rem' }}>
-                                {strategy.parameters?.rollout}%
-                            </Badge>
-                        }
-                    />
                 </StyledTitle>
-                {foundEnvironment && !addEditStrategy ? (
+                {foundEnvironment && (
```

Comment:
Nope, this is still wrong. Remove the entire scope. Even if foundEnvironment is true, we don't want to render anything here.